app/views/posts/index.html.erb

```diff
@@ -1 +1,22 @@
-<h1>Show all posts</h1>
\ No newline at end of file
+<section class="section">
+<% if @user.nil? %>
+  <h4>User not found</h4>
+<% else %>
+<div>
+  <%= render partial: 'users/user', locals: { user: @user } %>
+<div>
+<% end %>
+
+<ul>
+<% @posts.each_with_index do |post, index| %>
+  <li>
+  <%= render partial: 'users/post', locals: { post: post, index: index } %>
+  <% if post.comments_counter > 0 %>
+          <% post.five_most_recent_comments.each do |comment| %>
+            <%= render 'comments', comment: comment, user: @user %>
+            <% end %>
+          <% end %>
+  </li>
+  <% end %>
+</ul>
+</section>
```

Comment:
- [ OPTIONAL ] Comments should have its own border as on the given [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)

Yours:
![image](https://user-images.githubusercontent.com/62705947/236011962-ca3fdc90-3037-4cd9-b0f5-6ae12829906e.png)

Wireframe:
![image](https://user-images.githubusercontent.com/62705947/236012010-904d6bc5-55db-48f3-8954-c6b68ea18b2f.png)